src/containers/BookForm.js

```diff
@@ -1,20 +1,29 @@
-/* eslint-disable no-unused-vars */
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
-import { addBook } from '../actions';
+import { addBook } from '../actions/index';
 
 const BookForm = ({ addBook }) => {
   const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
   const categoriesOptions = categories.map(category => <option key={category}>{category}</option>);
 
   const [title, setTitle] = useState('');
   const [category, setCategory] = useState('');
+  const [id, setId] = useState(5);
+
```

Comment:
We also need to implement `handleChange` method.;
![image](https://user-images.githubusercontent.com/31897434/97777097-1d7cc200-1b6e-11eb-94e1-d1a51b590d12.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896828/